core/keyserver/keyserver.go

```diff
@@ -132,7 +136,7 @@ func (s *Server) getEntry(ctx context.Context, userID, appID string, firstTreeSi
 			LogId: s.logID,
 		})
 	if err != nil {
-		log.Printf("tlog.GetLatestSignedLogRoot(%v): %v", s.logID, err)
+		glog.Warningf("tlog.GetLatestSignedLogRoot(%v): %v", s.logID, err)
```

Comment:
Why is this a warning and others are errors? In general, how are we deciding if it's an error or a warning?
Same below in this file.

I would assume if the execution can continue it's a warning, otherwise it's an error. This one seems like an error to me. What do you think?